docs/advanced_usage.rst

```diff
@@ -170,12 +170,15 @@ To suppress the prompts asking for input, use `no_input`.
 Basic Example: Using the Defaults
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-TODO: document `no_input`:
+Cookiecutter will pick its defaults value if ussing with `no_input`::
 
-* As command-line argument
-* As parameter of `cookiecutter()`
+    from cookiecutter.main import cookiecutter
+    cookiecutter(
+        'cookiecutter-django',
+        no_input=True,
+    )
 
-TODO: document where context values come from in this example (`cookiecutter.json` and `.cookiecutterrc`)
+In this case it will be using the default defined in `cookiecutter.json` and `.cookiecutterrc`.
```

Comment:
I think we may need some concrete values here and to note that values from `.cookiecutterrc` will override values in `cookiecutter.json`.